src/data/nat/basic.lean

```diff
@@ -1258,6 +1278,9 @@ strict_mono.lt_iff_lt (pow_left_strict_mono k)
 lemma pow_left_injective {m : ℕ} (k : 1 ≤ m) : function.injective (λ (x : ℕ), x^m) :=
 strict_mono.injective (pow_left_strict_mono k)
 
+theorem pow_two_sub_pow_two (a b : ℕ) : a ^ 2 - b ^ 2 = (a + b) * (a - b) :=
```

Comment:
This one was added in #4135 but removed again in #4243, even though it isn't redundant - the general lemma needs a `comm_ring`.

Reply:
I missed this while reviewing #4243. Sorry.